src/Server/RestRouterHandlers/RestRouterHandler.h

```diff
@@ -76,6 +82,21 @@ class RestRouterHandler : private boost::noncopyable
         }
     }
 
+    static Poco::JSON::Object::Ptr parseRequest(HTTPServerRequest & request)
```

Comment:
Let's move this to anonymous namespace to reduce the header file size and possible pollution (compile time overhead) when things get changed